journal.py

```diff
@@ -56,26 +77,60 @@ def init_db():
     Base.metadata.create_all(engine)
 
 
+@view_config(route_name='newpost', renderer='templates/newpost.jinja2')
+def newpost(request):
+    return {}
+
+
 @view_config(route_name='home', renderer='templates/lists.jinja2')
 def list_view(request):
     entries = Entry.all()
     return {'entries': entries}
 
 
-@view_config(route_name='newpost', renderer='templates/newpost.jinja2')
+@view_config(route_name='details', renderer='templates/details.jinja2')
+def details(request):
+    entry = Entry.one(request.matchdict['id'])
```

Comment:
what about the possible error raised by using the `one` query api from SQLAlchemy?